packages/adobe_analytics/android/build.gradle

```diff
@@ -41,4 +41,7 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation "com.adobe.marketing.mobile:analytics:1.+"
```

Comment:
Is it possible to only have this one as a dependency of the library?
If we don't need imports of userprofile and sdk-core it can be a requirement for implementing apps